countrycodepicker/src/main/java/com/rilixtech/widget/countrycodepicker/CountryUtils.java

```diff
@@ -125,6 +125,8 @@ public static int getFlagDrawableResId(Country country) {
         return R.drawable.flag_croatia;
       case "cu": //cuba
         return R.drawable.flag_cuba;
+        case "cw": //cuba
```

Comment:
don't think the comment is right. probably unnecessary anyways as the drawable name is quite telling. Also format file please